frontend/app/src/composables/assets/spam.ts

```diff
@@ -0,0 +1,57 @@
+import { type ActionStatus } from '@/types/action';
+
+export const useSpamAsset = () => {
+  const { notify } = useNotificationsStore();
+  const { t } = useI18n();
+
+  const {
+    markAssetAsSpam: markAssetAsSpamCaller,
+    removeAssetFromSpamList: removeAssetFromSpamListCaller
+  } = useAssetSpamApi();
+
+  const { fetchWhitelistedAssets } = useWhitelistedAssetsStore();
+  const { fetchIgnoredAssets } = useIgnoredAssetsStore();
+
+  const markAssetAsSpam = async (token: string): Promise<ActionStatus> => {
+    try {
+      await markAssetAsSpamCaller(token);
+      await fetchWhitelistedAssets();
+      await fetchIgnoredAssets();
+      return { success: true };
+    } catch (e: any) {
+      notify({
+        title: t('ignore.spam.failed.mark_title').toString(),
+        message: t('ignore.spam.failed.mark_message', {
+          message: e.message
+        }).toString(),
```

Comment:
The `.toString()` call here is redundant we don't need it anymore since a while now.

So we can skip it.